src/simulator_core/infrastructure/app.py

```diff
@@ -33,7 +33,7 @@ def run(file_path: str = None):
                                      name="test run",
                                      timestep=3600,
                                      start=0,
-                                     stop=3600 * 48)
+                                     stop=3600 * 1)
```

Comment:
Comment sltaat niet op dit punt!! 

Er zit nog een pad in deze file willen we dat niet relatife maken? (print(run(r'd:\repos\NWN\simulator-core\testdata\test1.esdl'))
)

Reply:
Aangepast